src/accessory/thermostat-accessory.ts

```diff
@@ -132,7 +133,15 @@ export default class ThermostatAccessory extends BaseAccessory {
   }
 
   async handleTargetStateGet(): Promise<number> {
```

Comment:
I forgot to add the the try catch part for getState indeed, thanks for the update :), looks good to me